# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1423818871 -3600
#      Fri Feb 13 10:14:31 2015 +0100
# Node ID 18fd412c7dc3bc9238e6f938422c026735252ada
# Parent  ef3cdadc2a0825210d498e42f6e5c2a20ffa95e7
Remove erroneously added "keyboard" in cset ef3cdadc2a08

diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -425,17 +425,17 @@ function print (varargin)
 
     ## call the graphics toolkit print script
     switch (get (opts.figure, "__graphics_toolkit__"))
       case "gnuplot"
         opts = __gnuplot_print__ (opts);
       otherwise
         opts = __opengl_print__ (opts);
     endswitch
-keyboard
+
   unwind_protect_cleanup
     ## restore modified properties
     if (isstruct (props))
       ## Restore figure position and units first
       for n = 2:-1:1
         if (ishandle (props(n).h))
           set (props(n).h, props(n).name, props(n).value{1});
         endif
